Apps/Src/ReadSwitches.c

```diff
@@ -51,7 +51,7 @@ void Task_ReadSwitches(void* p_arg) {
         }
         
         // motor on/off
-        Contactors_Set(MOTOR_CONTACTOR, Switches_Read(IGN_2));
+        Contactors_Set(MOTOR_CONTACTOR, Switches_Read(IGN_2), SET_CONTACTORS_NON_BLOCKING);
```

Comment:
Shouldn't this be a blocking call? We don't want this to fail.

Reply:
Yes, you are so right! Thank you for noticing that.